app/views/shared/_service_comparison.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="slab">
   <div class="grid">
-    <div class="service-comparison">
+    <div class="service-comparison data-sticky-container">
```

Comment:
the sticky library looks for the class passed to new Sticky() instance _inside of_ the data-sticky-container. Data-sticky-container becomes the reference point for what the class you designate gets "stuck" to. It's kind of a weird implementation since a part of it is so explicit and the other is so inferred.